docker-compose-dev.yml

```diff
@@ -0,0 +1,318 @@
+# an ephemeral instance of ChRIS backend services for local development
+#
+# warning: /var/run/docker.sock is mounted into some services (notably pman)
+
+services:
+  chrisomatic:
+    image: ghcr.io/fnndsc/chrisomatic:0.8.2
+    profiles:
+      - tools
+    volumes:
+      - "./chrisomatic/chrisomatic.yml:/chrisomatic.yml:ro"
+      - "/var/run/docker.sock:/var/run/docker.sock:rw"
+    userns_mode: host
+    depends_on:
+      - chris
+    networks:
+      - local
+
+  db_migrate:
+    image: ghcr.io/fnndsc/cube:5.0.0
+    command:
+      - sh
+      - -c
+      - pip install -r /opt/ChRIS_ultron_backEnd/requirements/local.txt && python manage.py migrate --noinput
+    volumes:
+      - chris_files:/var/chris:rw
+      - ./chris_backend:/opt/app-root/src
+      - .:/opt/ChRIS_ultron_backEnd
+    environment:
+      - DJANGO_SETTINGS_MODULE=config.settings.local
+      - STORAGE_ENV=fslink
+    depends_on:
+      chris_dev_db:
+        condition: service_healthy
+    networks:
+      - local
+
+  chris:
+    container_name: chris
+    image: ghcr.io/fnndsc/cube:5.0.0
```

Comment:
Since many of these services specify `pip install -r ...` in their command, I would suggest using a docker-compose build.